config/runtime/boot/u-boot.jinja2

```diff
@@ -11,15 +11,15 @@
 {%- endif %}
 {%- if boot_commands == 'nfs' %}
     nfsrootfs:
-      url: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240221.0/{{ platform_config.debarch }}/full.rootfs.tar.xz'
+      url: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240221.0/{{ debarch }}/full.rootfs.tar.xz'
       compression: xz
     ramdisk:
-      url: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240221.0/{{ platform_config.debarch }}/initrd.cpio.gz'
+      url: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240221.0/{{ debarch }}/initrd.cpio.gz'
       compression: gz
     os: debian
 {%- else %}
     ramdisk:
-      url: 'http://storage.kernelci.org/images/rootfs/buildroot/buildroot-baseline/20230703.0/{{ platform_config.debarch }}/rootfs.cpio.gz'
```

Comment:
Shouldn't buildroot images use `karch` instead? Let me double check

Reply:
Dammit you're right, pushing a fix right now!